src/store/index.ts

```diff
@@ -1,25 +1,26 @@
 import { defineStore } from 'pinia';
 import { getAllMoviesData, getMovieGenresData, getSeancesData } from '@/services/api/data.js';
+import { GenreData, MovieData, SeanceData, ScreeningData } from '@/types/data';
 import router from '@/router';
 
 export const useMainStore = defineStore('main', {
   state() {
     return {
-      error: null,
-      errorDisplayTime: 4,
+      error: null as Error,
+      errorDisplayTime: 4 as number,
```

Comment:
Basic types will be inferred without issues 😎 
```suggestion
      errorDisplayTime: 4,
```

Reply:
Right, i've added it because TS had problems with `this` in store, but it's solved now as u know 😄